packages/grafana-data/src/datetime/moment_wrapper.ts

```diff
@@ -79,6 +79,10 @@ export const setLocale = (language: string) => {
   moment.locale(language);
 };
 
+export const getLocale = () => {
+  return moment.locale();
+};
+
```

Comment:
It would be better if we don't extend the moment_wrapper API with more functions but at the same time there is no better way of doing this.